src/app/content/components/Page/PageContent.tsx

```diff
@@ -150,6 +150,8 @@ export default styled(MainContent)`
 
   .os-figure,
   .os-figure:last-child {
+    table-layout: fixed;
```

Comment:
If I'm reading the spec right, won't this fix the column width, which could change how some content is displayed? I'm wondering if it could cause overflow issues. I'm also wondering if this is for render performance to help page load?

Reply:
If you go to [this page](https://rex-web-production.herokuapp.com/books/workplace-software-skills@c795d5f/pages/preface?archive=https%3A%2F%2Fopenstax.org%2Fapps%2Farchive-preview%2F20231109.173216) and shift-scroll, you will see the main content area scroll horizontally.
When you scroll to the bottom of the page, the images there load, and the main content width resets.